### DISCUSSION TITLE
Is SFTP supported ?

### DISCUSSION THREAD

**Discussion by jsduchene** [roleoroleo/yi-hack-Allwinner-v2#617] [Q&A] [answered]
Hello,

I'm trying to test the FTP Upload feature and I'm wondering if the SFTP protocol is supported ?

Thanks

**Comment by roleoroleo** [comment]
Yes, SFTP is supported.

**Comment by jsduchene** [comment]
Do you have an example on how to configure? Do we have to put the port (22) in the hostname ?
I've tried several config but it does not work

**Reply by roleoroleo** [reply, marked as answer]
Sorry, at the moment sftp is supported only as a server.
Not in the put.

**Reply by jsduchene** [reply]
Thanks ! It confirm all my tests.

If that's something you could consider for future release, it would be great !
